docs/data/system/getting-started/overview/overview.md

```diff
@@ -26,5 +26,5 @@ Learn more on [the `sx` prop page](/system/getting-started/the-sx-prop/).
 
 [MUI Base](/base/getting-started/overview/) is a library of "unstyled" React components, while MUI System is a set of utilities for quickly applying styles to those components (as well as other MUI component libraries like Material UI and Joy UI).
 
-MUI Base is a standalone component library, whereas the System is _supplemental_ in that it's designed to be paired with Base and other MUI components, as well as third-party components.
+MUI Base is a standalone component library, whereas the system is _supplemental_ in that it's designed to be paired with MUI Base and other MUI components, as well as third-party components.
```

Comment:
Why lowercase here? We wouldn't refer to MUI Base as "base," right? I would argue that we should take it a step further in the other direction to enforce "MUI System" as the correct proper noun.

Reply:
My thought is that it's either MUI System or the generic name, like the system design tokens. 
```suggestion
MUI Base is a standalone component library, whereas MUI System is _supplemental_ in that it's designed to be paired with MUI Base and other MUI components, as well as third-party components.
```